.github/workflows/release-workflow-dispatcher.yml

```diff
@@ -0,0 +1,284 @@
+          echo "mayastor_version=$(curl -s $GITHUB_API_URL/openebs/mayastor-extensions/refs/heads/develop/chart/Chart.yaml | yq '.version')" >> "$OUTPUT_FILE"
+
+      - name: Debug extracted versions
+        run: |
+          echo "Extracted LocalPV Provisioner Version: ${{ steps.extract-versions.outputs.localpv_provisioner_version }}"
+          echo "Extracted ZFS LocalPV Version: ${{ steps.extract-versions.outputs.zfs_localpv_version }}"
+          echo "Extracted LVM LocalPV Version: ${{ steps.extract-versions.outputs.lvm_localpv_version }}"
+          echo "Extracted Mayastor Version: ${{ steps.extract-versions.outputs.mayastor_version }}"
+
+      - name: Test the chart version updater script
+        run: |
+          nix-shell --pure --run "./scripts/helm/test-update-chart-version.sh" ./shell.nix
+
+      - name: Modify the chart version based on the branch name for develop
+        run: |
+          branch_name=${{ github.ref_name }}
+          nix-shell --pure --run "./scripts/helm/update-chart-version.sh --branch $branch_name --type develop --localpv-provisioner-version ${LOCALPV_PROVISIONER_VERSION} --zfs-localpv-version ${ZFS_LOCALPV_VERSION} --lvm-localpv-version ${LVM_LOCALPV_VERSION} --mayastor-version ${MAYASTOR_VERSION}" ./shell.nix
+        env:
+          LOCALPV_PROVISIONER_VERSION: ${{ steps.extract-versions.outputs.localpv_provisioner_version }}
+          ZFS_LOCALPV_VERSION: ${{ steps.extract-versions.outputs.zfs_localpv_version }}
+          LVM_LOCALPV_VERSION: ${{ steps.extract-versions.outputs.lvm_localpv_version }}
+          MAYASTOR_VERSION: ${{ steps.extract-versions.outputs.mayastor_version }}
+
+      - name: Create Pull Request to develop
+        id: cpr
+        uses: peter-evans/create-pull-request@v5
+        with:
+          base: develop
+          commit-message: "chore(ci): update helm chart versions and/or git submodules"
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          title: "Prepare develop branch on ${{ github.ref_name }} creation"
+          labels: |
+            update-develop-branch
+            automated-pr
+          draft: false
+          signoff: true
+          branch: "create-pull-request/patch-develop"
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  update_release_branch_after_release:
+    outputs:
+      localpv_provisioner_version: ${{ steps.extract-versions-from-tag.outputs.localpv_provisioner_version }}
+      zfs_localpv_version: ${{ steps.extract-versions-from-tag.outputs.zfs_localpv_version }}
+      lvm_localpv_version: ${{ steps.extract-versions-from-tag.outputs.lvm_localpv_version }}
+      mayastor_version: ${{ steps.extract-versions-from-tag.outputs.mayastor_version }}
+    runs-on: ubuntu-22.04
+    if: github.ref_type == 'tag' && github.event_name == 'push' && !contains(github.ref_name, 'rc')
```

Comment:
checking just for rc does not seem very robust, probably you want to check properly with the semvertool in a step, something like:

`[ $(semver get release $tag) != $tag ]`

Reply:
This is no longer required as am using the script. I have removed it.